avm/ptn/azd/aks-automatic-cluster/tests/e2e/defaults/main.test.bicep

```diff
@@ -0,0 +1,48 @@
+targetScope = 'subscription'
+
+metadata name = 'Using only defaults and use AKS Automatic mode'
+metadata description = 'This instance deploys the module with the set of automatic parameters.'
```

Comment:
Not sure - but is this description accurate? I want to say yes because I've seen the word `auto` somwhere, but wanted to double-check

Reply:
Yes, it is indeed a test of an AKS cluster. I'm guessing you're seeing "another" `auto` [here](https://github.com/Azure/bicep-registry-modules/blob/main/avm/res/container-service/managed-cluster/tests/e2e/automatic/main.test.bicep#L5). They have some overlap, but we want to implement a template in the `ptn` module based on the default AKS cluster.

For more details, you can get from this [comment](https://github.com/Azure/bicep-registry-modules/pull/3104#discussion_r1769021485).

Reply:
Thanks for elaborating :) Resolving